apps/core/lib/core/services/shell/client.ex

```diff
@@ -54,4 +52,10 @@ defmodule Core.Shell.Client do
       as -> Poison.decode(body, as: as)
     end
   end
+
+  defp handle_error({:ok, %HTTPoison.Response{body: body}}) do
+    Logger.error "shell setup error: #{body}"
```

Comment:
I think this will probably leak credentials in plain text in the logs.

Reply:
If I understand correctly this should only forward error messages returned by our `v1/setup` endpoint from the CLI. I don't think it contains any plain text credentials. This error is anyway already logged on the other side when the endpoint is called.